stac/harness/local.py

```diff
@@ -359,7 +363,7 @@ def _want_details(econf):
     kids = econf.settings.children
     has_intra_oracle = has.intra and (kids.intra.oracle or kids.inter.oracle)
     return (has_maxent and
-            ('mst' in econf.parser.key or 'astar' in econf.parser.key) and
+            any(k in econf.parser.key for k in ('mst', 'astar', 'ilp')) and
```

Comment:
A tuple works fine but it feels weird here. We don't want to pack together a handful of (possibly heterogeneous) values and iterate over each of them but to check against a set of possible decoders, hence the recommended use of set or rather frozenset to clearly state the intended semantics.